src/content/events_and_highlights.json

```diff
@@ -0,0 +1,128 @@
+{
+  "sections": [
+    {
+      "title": "Our Events and Highlights",
+      "content": [
+        {
+          "par": "AnitaB.org Open Source Events convenes members from all backgrounds, and levels of expertise throughout the year to bridge the diversity gap in Open Source development."
+        },
+        {
+          "par": "Our events aim to give members insights from industry experts and female-identifying leaders on a variety of topics promoting Open Source, and covering stages of Open Source development right from Introduction to Quality Assurance."
+        }
+      ]
+    },
+    {
+      "title": "Events",
+      "content": [
+        [
+          {
+            "Event Title": "Why I contribute to Open source?"
+          },
+          {
+            "Event Date": "July 4, 2020"
+          },
+          {
+            "Event Location": "Virtual"
+          },
+          {
+            "Event Timings": "1am (UTC)"
+          },
+          {
+            "Event description": [
+              {
+                "par": "Heard about Open Source, not sure what’s in it for you? Need a boost to begin?"
+              },
+              {
+                "par": "This event will give you an understanding of the motivation behind individuals who contribute to open source"
+              },
+              {
+                "par": "Open Source contributors with varying experiences will be there to share their journey and answer your questions!"
+              }
+            ]
+          },
+          {
+            "Event Highlights": "src\\assets\\events_and_highlights\\1.png"
```

Comment:
we will not be able to load it like that, please check how it was done in conent/*.js files with `require`. Thanks.